docs/index.html

```diff
@@ -13,72 +13,39 @@
+            .left .circle { left: 0; border-right-color: transparent !important; -webkit-transform: rotate(129deg); transform: rotate(129deg); -webkit-animation: left-spin 2s cubic-bezier(0.4, 0.0, 0.2, 1) infinite both; animation: left-spin 2s cubic-bezier(0.4, 0.0, 0.2, 1) infinite both;}
+            .right .circle { left: -100%; border-left-color: transparent !important; -webkit-transform: rotate(-129deg); transform: rotate(-129deg); -webkit-animation: right-spin 2s cubic-bezier(0.4, 0.0, 0.2, 1) infinite both; animation: right-spin 2s cubic-bezier(0.4, 0.0, 0.2, 1) infinite both;}
+
+            @keyframes fill-unfill-rotate { 12.5% { transform: rotate(135deg);  } 25%   { transform: rotate(270deg);  } 37.5% { transform: rotate(405deg);  }  50%   { transform: rotate(540deg);  } 62.5% { transform: rotate(675deg);  } 75% { transform: rotate(810deg);  } 87.5% { transform: rotate(945deg);  } to { transform: rotate(1080deg); }}
+            @keyframes container-rotate { to { transform: rotate(360deg) } }
+            @keyframes left-spin { from { transform: rotate(130deg); } 50% { transform: rotate(-5deg); } to { transform: rotate(130deg); } }
+            @keyframes right-spin { from { transform: rotate(-130deg); } 50% { transform: rotate(5deg); } to { transform: rotate(-130deg); }}
         </style>
-                <div id="pre-bootstrap">
-                    <div class="messaging">
-                        <h1>
-                            <img src="./assets/TriFlame.svg" width="24" height="41"><br><img src="./assets/CashmereAppLogo.svg" width="200" height="50">
-                        </h1>
-                        <div class="loader"></div>
-                    </div>
+        <div id="pre-bootstrap">
+            <svg version="1.1" id="triflame" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 11.1 18.5" style="enable-background:new 0 0 11.1 18.5;" xml:space="preserve"><style type="text/css"> .st0{fill:#FFFFFF;} </style> <path class="st0" d="M5.7,18.3c-0.6-0.3-1.1-0.7-1.5-1.2c-0.4-0.5-0.8-1.1-1-1.8c-0.2-0.7-0.4-1.4-0.4-2.2c0-4.6,3.8-7,3.4-11.6 c0-0.3,0-0.5-0.1-0.8C5.9,0.5,5.8,0.3,5.7,0.2c0.6,0.3,1.1,0.7,1.5,1.2c0.4,0.5,0.8,1.1,1,1.8c0.2,0.7,0.4,1.4,0.4,2.2 c0,3.9-3.6,6.8-3.6,11c0,0.4,0.1,0.8,0.2,1.1C5.3,17.8,5.4,18.1,5.7,18.3"/> <path class="st0" d="M9.1,18.1c-0.4-0.2-0.7-0.5-1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.4-0.2-0.9-0.2-1.5c0-3.1,2.8-4.8,2.4-8.7 c0.5,0.5,1,1.4,1.3,2C10.9,8.3,11,8.8,11,9.3c0,2.6-2.4,4.6-2.4,7.4c0,0.3,0,0.5,0.1,0.7C8.8,17.7,8.9,17.9,9.1,18.1"/> <path class="st0" d="M2,12.9c-0.4-0.2-0.7-0.5-1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2C0.2,10.4,0.1,10,0.1,9.4c0-3.1,2.8-4.8,2.4-8.7 c0.5,0.5,1,1.4,1.3,2C3.9,3.2,4,3.6,4,4.2c0,2.6-2.4,4.6-2.4,7.4c0,0.3,0,0.5,0.1,0.7C1.8,12.5,1.9,12.7,2,12.9"/> </svg>
```

Comment:
Inlined these SVGs to cut back on stuff going over the wire. My assumption is that these would show up sooner for people visiting the page for the first time.